sysapi/sysapi_util/Unmarshal_TPMU_CAPABILITIES.c

```diff
@@ -87,6 +87,11 @@ void Unmarshal_TPMU_CAPABILITIES(
 			Unmarshal_TPML_ECC_CURVE( sysContext, &capabilities->eccCurves );
 			break;
 #endif
+#ifdef TPM_CAP_VENDOR_PROPERTY
```

Comment:
It's kinda weird that it was already defined but it came straight from the table from the spec so the original author must have put it in the headers from completeness.